Sources/AppcuesKit/Presentation/Traits/TraitComposer.swift

```diff
@@ -16,12 +16,14 @@ internal protocol TraitComposing: AnyObject {
 @available(iOS 13.0, *)
 internal class TraitComposer: TraitComposing {
 
+    private weak var appcues: Appcues?
```

Comment:
is this used?

Reply:
Yes, the reference is passed into the `ExperienceStepViewModel` on L95 because it's needed as part of the custom component init (for `AppcuesExperiencePluginConfiguration` and `AppcuesExperienceActions`).

Reply:
doh, my bad, i misread the diff here with a collapsed section. thank you